Client/mods/deathmatch/logic/lua/CLuaArgument.cpp

```diff
@@ -371,6 +371,42 @@ CClientEntity* CLuaArgument::GetElement() const
     return CElementIDs::GetElement(ID);
 }
 
+bool CLuaArgument::GetAsString(SString& strBuffer) const
+{
+    switch (m_iType)
+    {
+        case LUA_TNIL:
+            strBuffer = "nil";
+            break;
+        case LUA_TBOOLEAN:
+            if (m_bBoolean)
+                strBuffer = "true";
+            else
+                strBuffer = "false";
+            break;
+        case LUA_TTABLE:
```

Comment:
Shouldn't we allow only string and number arguments?

Reply:
Because of the issue raised in Discord with arg parser, we cannot specify exact types to `std::unordered_map` when receiving the Lua arguments, we have to use `CLuaArgument` which accepts any value. So I think this is fine, if they supply any bad values (such as function, or userdata) then it's simply not included.

Reply:
Also this function is already exists, it's just copied from Server Deathmatch project.

Reply:
No longer relevant due to #3246